vcx/wrappers/java/vcx/src/main/java/com/evernym/sdk/vcx/issuer/IssuerApi.java

```diff
@@ -0,0 +1,269 @@
+        @SuppressWarnings({"unused", "unchecked"})
+        public void callback(int commandHandle, int err, long handle) {
+            CompletableFuture<Long> future = (CompletableFuture<Long>) removeFuture(commandHandle);
+            if (!checkCallback(future, err)) return;
+            // TODO complete with exception if we find error
+//            if (err != 0) {
+//                future.completeExceptionally();
+//            } else {
+//
+//            }
+            Long result = handle;
+            future.complete(result);
+        }
+    };
+
+    public static CompletableFuture<Long> issuerCredentialDeserialize(String serializedData) throws VcxException {
+        ParamGuard.notNull(serializedData, "serializedData");
+        CompletableFuture<Long> future = new CompletableFuture<>();
+        int issue = addFuture(future);
+
+        int result = LibVcx.api.vcx_issuer_credential_deserialize(
+                issue,
+                serializedData,
+                issuerCredentialDeserializeCB
+        );
+        checkResult(result);
+        return future;
+    }
+
+
+
+    public static CompletableFuture<Integer> issuerTerminateCredential(
+            int credentialHandle,
+            int state,
+            String msg
+    ) throws VcxException {
+        ParamGuard.notNull(credentialHandle, "credentialHandle");
+        ParamGuard.notNull(state, "state");
+        ParamGuard.notNullOrWhiteSpace(msg, "msg");
+        CompletableFuture<Integer> future = new CompletableFuture<>();
+        int issue = addFuture(future);
+
+        int result = LibVcx.api.vcx_issuer_terminate_credential(
+                issue,
+                credentialHandle,
+                state,
+                msg);
+        checkResult(result);
+
+        return future;
+
+    }
+    public static CompletableFuture<Integer> issuerCredntialRelease(
+            int credentialHandle
+    ) throws VcxException {
+        ParamGuard.notNull(credentialHandle, "credentialHandle");
+        CompletableFuture<Integer> future = new CompletableFuture<>();
+
+        int result = LibVcx.api.vcx_issuer_credential_release(credentialHandle);
+        checkResult(result);
+
+        return future;
+    }
+
+    public static CompletableFuture<Integer> issuercredentialRequest(
+            int credentialHandle,
+            String credentialRequest) throws VcxException {
+
+        ParamGuard.notNull(credentialHandle, "credentialHandle");
+        ParamGuard.notNull(credentialRequest, "credentialRequest");
+        CompletableFuture<Integer> future = new CompletableFuture<>();
+
+        int result = LibVcx.api.vcx_issuer_get_credential_request(
+                credentialHandle,
+                credentialRequest);
+        checkResult(result);
+
+        return future;
+    }
+
+    public static CompletableFuture<Integer> issuerAcceptRequest(
+            int credentialHandle) throws VcxException {
```

Comment:
Yes, I have noticed the inconsistency and refrained from fixing them as it will cause the beaking change in Connect.me.
I can do the refactoring if breaking change at pis level is not a big issue for connect.me

Reply:
I am not sure which breaking change are you talking about. I was referring to wrapping of brackets and curl braces